### ISSUE TITLE
Tapping Backpack icon no longer closes inventory window

### ISSUE THREAD

**Issue by lilwhitemouse** [00-Evan/shattered-pixel-dungeon#976]
It used to be that if you had the inventory open, if you pressed the backpack tab icon under the inventory, the window would close. Likewise, if you switched to the Velvet Pouch and then tapped the pouch icon under the window, it would close. These were very useful ways to close the windows on an android device! Could you bring that back?

Thanks!

**Comment by 00-Evan** [comment]
The button is part of the window, and so it was incorrect functionality for it to close the window. A tap on any other part of the inventory window doesn't close it. Sorry to hear you have mobility issues, but even at very high zoom the difference between tabbing on the tab and tapping below it should be about half a centimeter. You can also use the back button on your device to close the window.

**Comment by lilwhitemouse** [comment]
Ah.  Let me rephrase/restart this:
To increase accessibility for disabled people, would you implement functionality so that tapping the container icon/window icon closes the window automatically (as a toggle)?
Thank you

**Comment by lilwhitemouse** [comment]
I could open a new issue, if you would like?

**Comment by 00-Evan** [comment]
I'm fine with discussing this here.

I would rather not add a toggle for this functionality, If I added a toggle even half of the time players requested one then the settings menu would have hundreds of options by now. Also, I don't really understand the argument that this is an accessibility benefit, when you can get the exact same functionality by just tapping slightly lower on your screen so that the tap is actually below the window instead of over the already activated inventory tab.
